src/skins/vector/views/molecules/RoomTile.js

```diff
@@ -50,7 +50,16 @@ module.exports = React.createClass({
             'mx_RoomTile_highlight': this.props.highlight,
             'mx_RoomTile_invited': this.props.room.currentState.members[myUserId].membership == 'invite'
         });
-        var name = this.props.room.name.replace(":", ":\u200b");
+
+        var name;
+        if (this.props.isInvite) {
+            name = this.props.room.getMember(MatrixClientPeg.get().credentials.userId).events.member.getSender();
+        }
+        else {
+            name = this.props.room.name;
+        }
+
+        name = name.replace(":", ":\u200b");
```

Comment:
For us mortals who don't know unicode code points, it would be nice to explain what `\u200b` is and why we need it :angel: